examples/python/Quantinuum_compile_without_api.py

```diff
@@ -1,3 +1,6 @@
+# <div style="text-align: center;">
+# <img src="https://assets-global.website-files.com/62b9d45fb3f64842a96c9686/62d84db4aeb2f6552f3a2f78_Quantinuum%20Logo__horizontal%20blue.svg" width="200" height="200" /></div>
```

Comment:
I noticed this notebook still says...

```
It's possible that the hardcoded verion is not up to date with the latest native gate set as described in the [System Model H1 Product Data Sheet](https://www.quantinuum.com/hardware/h1). In this case, the Rzz gate, which is the ZZPhase gate in pytket, is missing. This can be added by running the following command.
```

Is this still needed?

I showed compile without API in this stack exchange post -> https://quantumcomputing.stackexchange.com/questions/34571/how-do-i-compile-a-circuit-to-the-native-gateset-of-quantinuums-trapped-ion-dev/34572#34572

Reply:
Ahh, that's nice! When this one was made, that wasn't the case, it wasn't updated in pytket-quantinuum. Going forward I think pytket-quantinuum should stay tied out with releases of new native gates, so I'll remove this. I know there's one coming, so will need to watch for that.